shared/chat/conversation/input-area/normal/platform-input.native.tsx

```diff
@@ -135,11 +135,11 @@ const Buttons = (p: ButtonsProps) => {
 const AnimatedIcon = createAnimatedComponent(Kb.Icon)
 const AnimatedExpand = (() => {
   if (skipAnimations) {
-    return React.memo(() => {
+    return React.memo(function AnimatedExpand() {
```

Comment:
this form is updated everywhere so we can see these component names, else in devtools it'll be `Unknown` which makes it hard to understand whats thrashing